### DISCUSSION TITLE
SSL activation nginx_app

### DISCUSSION THREAD

**Discussion by manish050994** [Sunbird-cQube/community#868] [General]
Dear Team i am using cqube v5.0.3, during the renewal of ssl I follow the flowing steps :
1. copy the new crt and key file into the nginx_app:etc/ss/<domain name>
2. change the file name in default.conf file
3. restart the container
note: crt and key is valid.
Issue: crt file in the container is different form crt file(same as old crt file) downloaded from the browser.
message i am getting in the browser: "This server could not prove that it is <domain name>; its security certificate expired 94 days ago." 

approach i tried to resolve the issue: 
1. Stop the container
2. remove the nginx_app container
3. docker-compose up the nginx_app container
4. copy the crt and key file again in the nginx_app container
5. change the file name in default.conf file 
6. restart the nginx_app container

But the issue is same as before.

checked in different browsers by deleting the cookies.

**Comment by PavitraIR** [comment]
@manish050994 

Please check the cert and key are matching or not, both cert and key should be match , if yes....Please find the below steps for nginx_ms

  1. cd /opt/cqube/nginx
  2. ls
  3. cat default.conf
  Note: Check the all port and Crt and key and domain name


Then copy the new crt and key files in nginx container(from your local to container):

  1. sudo docker exec -it niginx_app bash
  2. ls
  3. cd etc/ssl
  4. ls
  5. Create one domain directory(example :-- mkdir <domain_name>)
  6. cd <domain_name>
  7. cd
  8. cd ssl_certificates
  9. sudo docker cp server.crt nginx_app:/etc/ssl/<domain_name>
  10. sudo docker cp server.key nginx_app:/etc/ssl/<domain_name>
  11. sudo docker restart nginx_app

sudo docker logs nginx_app (Then check the logs of nginx)
